Makefile

```diff
@@ -46,10 +46,16 @@ build-static:
 push-image:
 	docker push $(IMAGE_NAME):$(VERSION)
 
-test: codegen download-test-deps
+unit-test:
+	go test -v -count=1 -cover -timeout 2h ./...
```

Comment:
Doesn't this include ./test anyway?

Reply:
Also: why disable caching?

Reply:
Disable caching? Not sure I follow?

Reply:
`-count=1` is the incanation for disabling the go test cache.

Reply:
Ohh I see now. Probably just copy/paste from other repos. For CI I'd want them to run each time, but for local caching would be nice. I can remove.